src/components/Registration/BusinessTypeConfirm.vue

```diff
@@ -53,14 +53,14 @@ export default class BusinessTypeConfirm extends Vue {
   protected checked = false
   protected label = 'General Partnership'
   protected labelSP = 'BC Sole Proprietorship / Doing Business As (DBA) Registration'
-  protected text = `I acknowledge that a General Partnership cannot be \
-    changed into a Sole Proprietorship (including DBA). If this is \
```

Comment:
The latest ES Lint tried to reformat this in a way that's incompatible with our current framework so I manually reformatted this into something ES Lint likes.